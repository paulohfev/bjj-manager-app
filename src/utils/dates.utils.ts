import { DateFormat } from '@/constants/dateFormats'

import dayjs from '../dates'

export const formatDate = (date: string): string => {
  return dayjs(date).format(DateFormat.DD_MM_YYYY_slash)
}
