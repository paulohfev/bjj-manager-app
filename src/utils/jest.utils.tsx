import React from 'react'

import { ThemeOptions, ThemeProvider, createTheme } from '@mui/material'

import { render } from '@testing-library/react'

import muiThemeConfig from '@/theme/mui-theme-config'

export const renderWithTheme = (component: React.ReactNode) => {
  return render(<ThemeProvider theme={createTheme(muiThemeConfig as ThemeOptions)}>{component}</ThemeProvider>)
}
