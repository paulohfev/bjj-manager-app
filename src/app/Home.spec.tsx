import Home from './page'
import '@testing-library/jest-dom'
import { renderWithTheme } from '@/utils/jest.utils'

describe('Home', () => {
  it('should match the snapshot', () => {
    const component = renderWithTheme(<Home />)
    expect(component).toMatchSnapshot()
  })
})
