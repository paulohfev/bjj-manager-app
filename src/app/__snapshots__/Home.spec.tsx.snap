// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-0"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-0"
            >
              <h1
                class="MuiTypography-root MuiTypography-body1 css-sbn2d4-MuiTypography-root"
              >
                2024
              </h1>
              <h3
                class="MuiTypography-root MuiTypography-body1 css-1i5uads-MuiTypography-root"
              >
                May
              </h3>
              <table
                class="MuiTable-root css-rqglhn-MuiTable-root"
              >
                <thead
                  class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
                >
                  <tr
                    class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                  >
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Sun
                    </th>
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Mon
                    </th>
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Tue
                    </th>
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Wed
                    </th>
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Thu
                    </th>
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Fri
                    </th>
                    <th
                      class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                      scope="col"
                    >
                      Sat
                    </th>
                  </tr>
                </thead>
                <tbody
                  class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
                >
                  <tr
                    class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                  >
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    />
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    />
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    />
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        01
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        02
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        03
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        04
                      </div>
                    </td>
                  </tr>
                  <tr
                    class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                  >
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        05
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        06
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        07
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        08
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        09
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        10
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        11
                      </div>
                    </td>
                  </tr>
                  <tr
                    class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                  >
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        12
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        13
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        14
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        15
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        16
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        17
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        18
                      </div>
                    </td>
                  </tr>
                  <tr
                    class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                  >
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        19
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        20
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        21
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        22
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        23
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        24
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        25
                      </div>
                    </td>
                  </tr>
                  <tr
                    class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                  >
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        26
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        27
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        28
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        29
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-9f1ieb"
                      >
                        30
                      </div>
                    </td>
                    <td
                      class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                    >
                      <div
                        class="MuiBox-root css-qjy1iy"
                      >
                        31
                      </div>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique unde fugit veniam eius, perspiciatis sunt? Corporis qui ducimus quibusdam, aliquam dolore excepturi quae. Distinctio enim at eligendi perferendis in cum quibusdam sed quae, accusantium et aperiam? Quod itaque exercitationem, at ab sequi qui modi delectus quia corrupti alias distinctio nostrum. Minima ex dolor modi inventore sapiente necessitatibus aliquam fuga et. Sed numquam quibusdam at officia sapiente porro maxime corrupti perspiciatis asperiores, exercitationem eius nostrum consequuntur iure aliquam itaque, assumenda et! Quibusdam temporibus beatae doloremque voluptatum doloribus soluta accusamus porro reprehenderit eos inventore facere, fugit, molestiae ab officiis illo voluptates recusandae. Vel dolor nobis eius, ratione atque soluta, aliquam fugit qui iste architecto perspiciatis. Nobis, voluptatem! Cumque, eligendi unde aliquid minus quis sit debitis obcaecati error, delectus quo eius exercitationem tempore. Delectus sapiente, provident corporis dolorum quibusdam aut beatae repellendus est labore quisquam praesentium repudiandae non vel laboriosam quo ab perferendis velit ipsa deleniti modi! Ipsam, illo quod. Nesciunt commodi nihil corrupti cum non fugiat praesentium doloremque architecto laborum aliquid. Quae, maxime recusandae? Eveniet dolore molestiae dicta blanditiis est expedita eius debitis cupiditate porro sed aspernatur quidem, repellat nihil quasi praesentium quia eos, quibusdam provident. Incidunt tempore vel placeat voluptate iure labore, repellendus beatae quia unde est aliquid dolor molestias libero. Reiciendis similique exercitationem consequatur, nobis placeat illo laudantium! Enim perferendis nulla soluta magni error, provident repellat similique cupiditate ipsam, et tempore cumque quod! Qui, iure suscipit tempora unde rerum autem saepe nisi vel cupiditate iusto. Illum, corrupti? Fugiat quidem accusantium nulla. Aliquid inventore commodi reprehenderit rerum reiciendis! Quidem alias repudiandae eaque eveniet cumque nihil aliquam in expedita, impedit quas ipsum nesciunt ipsa ullam consequuntur dignissimos numquam at nisi porro a, quaerat rem repellendus. Voluptates perspiciatis, in pariatur impedit, nam facilis libero dolorem dolores sunt inventore perferendis, aut sapiente modi nesciunt.
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-0"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-0"
          >
            <h1
              class="MuiTypography-root MuiTypography-body1 css-sbn2d4-MuiTypography-root"
            >
              2024
            </h1>
            <h3
              class="MuiTypography-root MuiTypography-body1 css-1i5uads-MuiTypography-root"
            >
              May
            </h3>
            <table
              class="MuiTable-root css-rqglhn-MuiTable-root"
            >
              <thead
                class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
              >
                <tr
                  class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Sun
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Mon
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Tue
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Wed
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Thu
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Fri
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-2zgg7m-MuiTableCell-root"
                    scope="col"
                  >
                    Sat
                  </th>
                </tr>
              </thead>
              <tbody
                class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
              >
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  />
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  />
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  />
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      01
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      02
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      03
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      04
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      05
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      06
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      07
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      08
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      09
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      10
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      11
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      12
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      13
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      14
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      15
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      16
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      17
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      18
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      19
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      20
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      21
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      22
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      23
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      24
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      25
                    </div>
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      26
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      27
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      28
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      29
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-9f1ieb"
                    >
                      30
                    </div>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-u40oc-MuiTableCell-root"
                  >
                    <div
                      class="MuiBox-root css-qjy1iy"
                    >
                      31
                    </div>
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique unde fugit veniam eius, perspiciatis sunt? Corporis qui ducimus quibusdam, aliquam dolore excepturi quae. Distinctio enim at eligendi perferendis in cum quibusdam sed quae, accusantium et aperiam? Quod itaque exercitationem, at ab sequi qui modi delectus quia corrupti alias distinctio nostrum. Minima ex dolor modi inventore sapiente necessitatibus aliquam fuga et. Sed numquam quibusdam at officia sapiente porro maxime corrupti perspiciatis asperiores, exercitationem eius nostrum consequuntur iure aliquam itaque, assumenda et! Quibusdam temporibus beatae doloremque voluptatum doloribus soluta accusamus porro reprehenderit eos inventore facere, fugit, molestiae ab officiis illo voluptates recusandae. Vel dolor nobis eius, ratione atque soluta, aliquam fugit qui iste architecto perspiciatis. Nobis, voluptatem! Cumque, eligendi unde aliquid minus quis sit debitis obcaecati error, delectus quo eius exercitationem tempore. Delectus sapiente, provident corporis dolorum quibusdam aut beatae repellendus est labore quisquam praesentium repudiandae non vel laboriosam quo ab perferendis velit ipsa deleniti modi! Ipsam, illo quod. Nesciunt commodi nihil corrupti cum non fugiat praesentium doloremque architecto laborum aliquid. Quae, maxime recusandae? Eveniet dolore molestiae dicta blanditiis est expedita eius debitis cupiditate porro sed aspernatur quidem, repellat nihil quasi praesentium quia eos, quibusdam provident. Incidunt tempore vel placeat voluptate iure labore, repellendus beatae quia unde est aliquid dolor molestias libero. Reiciendis similique exercitationem consequatur, nobis placeat illo laudantium! Enim perferendis nulla soluta magni error, provident repellat similique cupiditate ipsam, et tempore cumque quod! Qui, iure suscipit tempora unde rerum autem saepe nisi vel cupiditate iusto. Illum, corrupti? Fugiat quidem accusantium nulla. Aliquid inventore commodi reprehenderit rerum reiciendis! Quidem alias repudiandae eaque eveniet cumque nihil aliquam in expedita, impedit quas ipsum nesciunt ipsa ullam consequuntur dignissimos numquam at nisi porro a, quaerat rem repellendus. Voluptates perspiciatis, in pariatur impedit, nam facilis libero dolorem dolores sunt inventore perferendis, aut sapiente modi nesciunt.
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
